pinot-query-runtime/src/main/java/org/apache/pinot/query/service/QueryConfig.java

```diff
@@ -22,6 +22,9 @@
  * Configuration for setting up query runtime.
  */
 public class QueryConfig {
+  public static final int DEFAULT_MAX_INBOUND_QUERY_DATA_BLOCK_BYTES_SIZE = 128 * 1024 * 1024;
+  public static final long DEFAULT_TIMEOUT_NANO = 10_000_000_000L;
```

Comment:
Are we planning to make these configurable ?

Reply:
cc @walterddr

Reply:
yes we will make these configurable. 

but not for timeout, which will be take in per query